src/DebugEngineHost.Stub/DebugEngineHost.ref.cs

```diff
@@ -197,56 +182,131 @@ public object GetCustomLauncher(string launcherTypeName)
         {
             throw new NotImplementedException();
         }
-}
+    }
 
-/// <summary>
-/// The host logger returned from HostConfigurationStore.GetLogger.
-/// </summary>
-public sealed class HostLogger
+    /// <summary>
+    /// Level of logging used for HostLogChannel
+    /// </summary>
+    public enum LogLevel
```

Comment:
>LogLevel [](http://example.com/codeflow?start=16&length=8)

I wonder if we want to have fewer log levels than this? We need some equivalent to 'Error', 'Warning' and 'Verbose', but do we need quite this many?

Reply:
I was basing the log levels on MS.Logging package but we can reduce it down to the ones we use in VS for Natvis specifically

![image](https://user-images.githubusercontent.com/3953714/192650502-4a49faa8-6451-4a21-8f3d-a4b3e7399ea5.png)

Reply:
Here is my two cents worth, which you are free to ignore: I generally find that it is hard to choose appropriate log levels, and the difficulty increases as we add more levels.

Reply:
The mapping will be
```
Trace, Debug, Information => Verbose
Warning => Warning
Error => Error
None => None
```